packages/dds/tree/package.json

```diff
@@ -16,6 +16,8 @@
   "module": "lib/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "bench": "npm run clean && npm run tsc && npm run build:test && mocha --timeout 999999 -r node_modules/@fluidframework/mocha-test-setup --perfMode --fgrep @Benchmark --reporter \"../../../node_modules/@fluid-tools/benchmark/dist/MochaReporter.js\" \"./dist/test/**/*.js\"",
```

Comment:
seems odd to do a clean build here. We should trust the incremental build.

Also, I think this should work like "test" where we assume it's already built (after all, we assume all transitive deps have already been build, and all our test scripts also assume this).

If you really wanted to have a build and test script, I'd recommend making it a separate script, not doing a clean build, and incoking the build with fluid build so it will handle rebuilding deps of needed.

Reply:
Fixed, thx.